plugins/ci/pkg/ci/ci.go

```diff
@@ -109,19 +118,22 @@ func GetImagesFromManifest(configFolder string) ([]models.Image, []Resource, err
 	return images, resources, err
 }
 
-func processYamlNode(yamlNode map[string]interface{}) []models.Image {
+func processYamlNode(yamlNode map[string]interface{}) ([]models.Image, []Container) {
 	owner := models.Resource{
 		Kind: yamlNode["kind"].(string),
 		Name: yamlNode["metadata"].(map[string]interface{})["name"].(string),
 	}
 	podSpec := GetPodSpec(yamlNode)
 	images := getImages(podSpec.(map[string]interface{}))
-	return funk.Map(images, func(s string) models.Image {
+	return funk.Map(images, func(c Container) models.Image {
 		return models.Image{
-			Name:  s,
-			Owner: owner,
+			Name: c.Image,
+			Owner: models.Resource{
+				Kind: owner.Kind,
+				Name: fmt.Sprintf("%s/%s", owner.Name, c.Name),
```

Comment:
Do we need to put the container name in the `Owner`? I feel like `Owner` refs should reliably map to k8s resources...

Reply:
This is keeping it consistent with how the Trivy scan works right now. It uses the "Controller/container" name that gets saves in Insights.

Reply:
OK. We have to get rid of this pattern before it gets too pervasive 😓